blueprint/overview.md

```diff
@@ -163,6 +163,16 @@ Collaboration is initially controlled by lists of allowed domains. Individual us
 
 This blueprint does not include design information for a Secure Internet Gateway (SIG). A SIG is listed as a requirement in the Protective Security Policy Framework (PSPF) [Robust ICT Systems](https://www.protectivesecurity.gov.au/information/robust-ict-systems/Pages/default.aspx). 
 
+### Zero Trust security model
+
+"Zero Trust" is an IT security concept where the organisation does not automatically trust the user, the device they're on, or the network location they're connecting from - just because they are a known entity.
```

Comment:
I'm not sure how this bit connects with the sentence? Is it needed? If needed, probably separate into a separate sentence.

`just because they are a known entity.`

Reply:
I'll remove that as the statement "does not automatically" explains it.